File: WebServiceOfficePrivate/src/php/IMessageFromParent.php
<?php

class imessagefromparent{
    private $proxy;

    public function __construct($pUrl){
        $this->proxy = new SoapClient($pUrl);
    }

    public function setMessageAsReaded($messageId){
        $userData->messageId=new SoapVar($messageId, XSD_INT);
        return $this->proxy->__soapCall("setMessageAsReaded", array($userData) );
    }
    public function getNewMessages($userId){
        $userData->userId=new SoapVar($userId, XSD_INT);
        return $this->proxy->__soapCall("getNewMessages", array($userData) );
    }
    public function sendMessageToAdmin($idNatPrs, $titleMessage, $textMessage){
        $userData->idNatPrs=new SoapVar($idNatPrs, XSD_INT);
        $userData->titleMessage=new SoapVar(urlencode($titleMessage), XSD_STRING);
        $userData->textMessage=new SoapVar(urlencode($textMessage), XSD_STRING);
        return $this->proxy->__soapCall("sendMessageToAdmin", array($userData) );
    }
}

?>

File: WebServiceOfficePrivate/src/java/bonclub/office_private/web_service/IMessageFromParent.java
package bonclub.office_private.web_service;

import bonclub.office_private.web_service.common_objects.ParentMessage;
import bonclub.office_private.web_service.result.WsReturnValue;

import javax.jws.WebMethod; 
import javax.jws.WebParam; 
import javax.jws.WebService; 

@WebService
public interface IMessageFromParent{
    @WebMethod
    public WsReturnValue<Boolean> setMessageAsReaded(
        @WebParam(name="messageId") Integer messageId);
    @WebMethod
    public WsReturnValue<ParentMessage[]> getNewMessages(
        @WebParam(name="userId") Integer[] userId);
    @WebMethod
    public WsReturnValue<Integer> sendMessageToAdmin(
        @WebParam(name="idNatPrs") int idNatPrs, 
        @WebParam(name="titleMessage") String titleMessage, 
        @WebParam(name="textMessage") String textMessage);

}